aten/src/ATen/core/tensor_type.cpp

```diff
@@ -201,7 +201,7 @@ TensorTypePtr TensorType::create(const at::Tensor& t) {
   VaryingShape<size_t> stride_indices;
   VaryingShape<int64_t> strides;
   VaryingShape<int64_t> sizes;
-  if (!t.is_mkldnn() && !t.is_sparse() && !t.is_sparse_csr()) {
+  if (!t.is_mkldnn() && !t.is_sparse() && !t.is_sparse_csr() && !t.is_sparse_csc()) {
```

Comment:
At this point it seems more prudent to compare `t.layout() == c10::kStrided`

Reply:
We probably also eventually want a Python level `is_strided` as an equivalent to `is_sparse` and `is_sparse_csr`

Reply:
Yes, I totally agree. In fact, I'd like to introduce `is_strided` asap, say, as a preliminary PR within this ghstack.

Reply:
I expect that this will incur some level of discussion, so it's probably efficient to send it separately right now